iceoryx_hoofs/cmake/IceoryxPackageHelper.cmake

```diff
@@ -18,6 +18,7 @@
 # setup_package_name_and_create_files : this macro which is called from other modules which use iceoryx_hoofs
 # sets the variables for package version file,config file used for configuration
 # this also creates the config files
+include(GNUInstallDirs)
```

Comment:
Does this also work on MacOS and Windows? I am asking since we are supporting those platforms.

Reply:
Yes, it does. By default, the module uses the GNU conventions, which happen to coincide with the previous hard-coded values. The special cases such as multi-arch lib locations are only triggered for system-wide installation to prefix `/usr` or `/`, and only on certain Linux distributions which support these.